Workbooks/Virtual Machines/Availability/Server Availability.workbook

```diff
@@ -182,22 +181,17 @@
               "columnMatch": "$gen_group",
               "formatter": 13,
               "formatOptions": {
-                "linkTarget": "Resource",
+                "linkTarget": null,

```

Comment:
did changing this fix something or was it unintentional?  do individual computer rows still show up as links?

Reply:
unintentional, that did break the link good catch. fixed and resubmitting